client/ayon_core/modules/deadline/deadline_module.py

```diff
@@ -45,7 +45,8 @@ def get_plugin_paths(self):
         }
 
     @staticmethod
-    def get_deadline_pools(webservice, log=None):
+    def get_deadline_pools(webservice, auth=None, log=None):
+        # type: (str) -> list
```

Comment:
This should preferably be more specific in the return type, e.g. `List[str]`
This is lacking the second and third argument.
```suggestion
```
It's best to remove it since the docstring also provides type hints (which actually is incomplete with the new argument added.)

Should be e.g.:
```
        Args:
            webservice (str): Deadline Webservice URL.
            auth (Optional[Tuple[str, str]]): Tuple containing username, password
            log (Optional[Logger]): Logger to log errors to, if provided.
        Returns:
            List[str]: Pools.
```